### PR TITLE
Improve error message and add issue templates.

### PR DESCRIPTION
**Pull request by elijahtsai**
I have changed some error messages to be more informative. what do you think? because I still don't think it's clear enough for people to understand the message I want to tell. 🤔

Add lower version request template to let people request if they really need to use specific version.

Add higher version report to let people notify us there is a new version just updated.

### CODE REVIEW

File: sketchcrapp.sh [hunk 1]
```diff
@@ -268,15 +268,16 @@ analysisApplication() {
...
+    echo "[INFO] Copy the details below and open a new issue on GitHub repository: \
 https://github.com/duraki/SketchCrapp"
     echo "+==================================================================="
     echo "+ Issue details ‹s:sketchcrapp›"
     echo "+ Application Path  : $appPath"
     echo "+ Application Binary: $execPath"
     echo "+ Passed version    : $bundleVersionString"
     echo "+ Binary SHA1       : $appSHA1"
-    echo "+ Error             : Version not supported, review README file and try again."
+    echo "+ Error             : Version not supported."
```
Comment (duraki) [inline sketchcrapp.sh:280]: I'd go with: `Version $bundleVersionString is not supported.`

File: sketchcrapp.sh [hunk 2]
```diff
@@ -312,16 +313,18 @@ https://github.com/duraki/SketchCrapp"
...
 https://github.com/duraki/SketchCrapp"
       echo "+==================================================================="
-      echo "+ Issue details ‹s:sketchcrapp›"
       echo "+ Application Path  : $appPath"
       echo "+ Application Binary: $execPath"
       echo "+ Passed version    : $bundleVersionString"
       echo "+ Correct hash      : $(getHashFromVersionString "$bundleVersionString")"
       echo "+ Binary SHA1       : $appSHA1"
-      echo "+ Error             : Can’t look up version from hash."
+      echo "+ Error             : Hash invaild"
```
Comment (duraki) [inline sketchcrapp.sh:327]: I would leave it as `Can’t look up version from hash.`. It's more informative then just "Hash Invalid" (doesn't make sense).

Comment (duraki) [reply sketchcrapp.sh:327]: Or better yet; `Can't find Sketch with that signature. Hash is invalid.`

**Comment by elijahtsai** [comment]
I've tested every possible scenario, merged.

**Pull request merged**